src/scss/components/optional/_cuisines-widget.scss

```diff
@@ -0,0 +1,127 @@
+*
+* Example: Homepage – Popular cuisines.
+*
+* The `c-cuisinesWidget` component is an optional mixin within Fozzie.
+* If you'd like to use it in your project you can include it by adding `@include cuisinesWidget();` into your SCSS dependencies file.
+*
+* Documentation:
+* https://fozzie.just-eat.com/styleguide/ui-components/cuisines-widget
+*/
+
+$cuisinesWidget-titleColour: $white;
+$cuisinesWidget-defaultBackground: $brand--red;
+$cuisinesWidgetGrid-breakpoint-narrow: 570px;
+$cuisinesWidgetGrid-breakpoint-wide: 1241px;
+$cuisinesWidget-boxShadow: 0 2px 4px 0 rgba(0, 0, 0, 0.1);
+$cuisinesWidget-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0 rgba(0, 0, 0, 0.04);
+$cuisinesWidget-gradient: linear-gradient(to top, rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0));
+
+@mixin cuisinesWidget() {
+
+   .c-cuisinesWidget {
+       display: block;
+       border-radius: 8px;
+       overflow: hidden;
+       flex: 0 0 auto;
+       position: relative;
+       background-color: $cuisinesWidget-defaultBackground;
+       box-shadow: $cuisinesWidget-boxShadow;
+
+       &:hover {
+           box-shadow: $cuisinesWidget-hoverBoxShadow;
+       }
+
+       .c-cuisinesWidget-name {
+           position: absolute;
+           bottom: 0;
+           left: 0;
+           width: 100%;
+           padding: spacing(x2);
+           color: $cuisinesWidget-titleColour;
+           text-align: center;
+           background-image: $cuisinesWidget-gradient;
+
+           @media screen and (min-width: $cuisinesWidgetGrid-breakpoint-wide) {
+               @include font-size(large, false);
+               padding: spacing(x3);
+           }
+       }
+
+       .c-cuisinesWidget-img {
+           vertical-align: bottom;
+           width: 100%;
+       }
+
+       // grid of three elements for desktop, two for tablets and one for mobile
+       &.c-cuisinesWidget--gridOfThree {
+           min-height: 96px;
+           width: 100%;
+           max-width: 360px;
+           margin: auto;
+           margin-bottom: spacing(x2);
+
+           &:last-child {
+               margin-bottom: 0;
+           }
+
+           @include media('>=mid') {
+               min-height: 133px;
+               max-width: none;
+           }
+
+           @media screen and (min-width: $cuisinesWidgetGrid-breakpoint-narrow) {
+               width: calc(50% - 8px);
+               margin-right: spacing(x2);
+               margin-left: 0;
+               margin-top: 0;
+
+               &:nth-child(even) {
+                   margin-right: 0;
+               }
+
+               &:nth-last-child(2) {
+                   margin-bottom: 0;
+               }
+           }
+
+           @media screen and (min-width: $cuisinesWidgetGrid-breakpoint-wide) {
+               min-height: 145px;
+               width: calc((100% - 32px)/3);
+
+               &:nth-child(even) {
+                   margin-right: spacing(x2);
+               }
+
+               &:nth-child(3n+3),
```

Comment:
Always good to comment these `nth-child` selectors for what scenario they are targeting (just to make it super clear).

Reply:
✅